query/physicalplan/aggregate.go

```diff
@@ -717,6 +719,8 @@ func (a *Int64MinAggregation) Aggregate(pool memory.Allocator, arrs []arrow.Arra
 	switch typ {
 	case arrow.INT64:
 		return minInt64arrays(pool, arrs), nil
+	case arrow.FLOAT64:
```

Comment:
Instead of creating a `Float64MinAggregation` I changed this one to handle float as well. For the `MaxAggregation` I created a separate struct. I prefer having one function that can all both types but I don't know this code base so I'm happy to change it.

Reply:
Right. I wanted to check which was the better way.

Reply:
Agreed. I was looking at this very same thing earlier for uint64. I think having the switch in the aggregation function is better in the long run. 
Sadly, generics are still not feasible with the arrow library... :confused: